File: Datoteke/Datoteke.py
import time

def write_to_file(text, filename):
  for i in range(100000):
    with open(filename, "a") as temp:
        temp.write(text)

startTime = time.process_time()
write_to_file("To je besedilo, ki ga bomo zapisali v datoteko.\n", "izhod.txt")
endTime = time.process_time()

timeTaken = (endTime - startTime) * 1000.0
print(f"Execution time: {timeTaken:.3f} milliseconds")

File: Datoteke/Datoteke.go
package main

import (
    "fmt"
    "time"
	"os"
)

func writeToFile(text, filename string) {
  for i := 0; i < 100000; i++ {
	 os.WriteFile(filename, []byte(text), 0644)
  }
}

func main() {
  startTime := time.Now()
  writeToFile("To je besedilo, ki ga bomo zapisali v datoteko.\n", "izhod.txt")
  endTime := time.Now()
  timeTaken := endTime.Sub(startTime).Milliseconds()
  fmt.Printf("Time taken: %d milliseconds\n", timeTaken)
}